unlock-app/package.json

```diff
@@ -75,8 +75,9 @@
   },
   "engineStrict": true,
   "scripts": {
-    "dev": "npm run start-ganache && npm run build-paywall && npm run deploy-unlock-contract && nodemon src/server.js",
```

Comment:
can we move the `npm run start-ganache && npm run build-paywall && npm run deploy-unlock-contract` into a new `predev` script? this line is getting kind of longish.

Reply:
To clarify, you mean like:

```
"predev": "npm run start-ganache && npm run build-paywall && npm run deploy-unlock-contract,
"dev": "npm run predev && npm run build-blog" && nodemon src/server.js"
```

Reply:
```
  "predev": "npm run start-ganache && npm run build-paywall && npm run deploy-unlock-contract && npm run build-blog",
  "dev": "nodemon src/server.js"
```

npm auto-runs a thing with `pre` in front of the thing, so `predev` runs before `dev`, `prebuild` runs before `build`, etc.

Worth verifying before you push a commit of course, it might be `pre-dev` but I think it's no spa ce :)

The idea is that dev is only the running of the server, and the other stuff just makes dev possible

Reply:
Aha. I didn't know that. Thanks!